### PR TITLE
Add eiffel-graphql-api team to contacts

### PR DESCRIPTION
**Pull request by fdegir**
### Applicable Issues

N/A

### Description of the Change

This PR includes eiffel-graphql-api maintainer team in contacts.

### Alternate Designs

N/A

### Benefits

eiffel-graphql-api maintainer team is listed in contacts.

### Possible Drawbacks

N/A

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Fatih Degirmenci fatih.degirmenci@est.tech

### CODE REVIEW

**Comment by fdegir** [comment]
@t-persson @fredjn I wasn't sure if there is a maillist for eiffel-graphql-api team so I left that as not available. Please update the PR if there is one.

File: CONTACT.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Repository | Maintainer Team | Report Breach
 [eiffel-gerrit-herald](https://github.com/eiffel-community/eiffel-gerrit-herald) | [eiffel-gerrit-herald-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-gerrit-herald-maintainers) | Not available
 [eiffel-gerrit-lib ](https://github.com/eiffel-community/eiffel-gerrit-lib) | [eiffel-gerrit-herald-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-gerrit-herald-maintainers) | Not Available
 [eiffel-gerrit-plugin](https://github.com/eiffel-community/eiffel-gerrit-plugin) | [eiffel-gerrit-plugin-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-gerrit-plugin-maintainers) | [eiffel-gerrit-plugin-maintainers@googlegroups.com](mailto:eiffel-gerrit-plugin-maintainers@googlegroups.com)
+[eiffel-graphql-api](https://github.com/eiffel-community/eiffel-graphql-api) | [eiffel-graphql-api](https://github.com/orgs/eiffel-community/teams/eiffel-graphql-api) | Not Available
```
Comment (t-persson) [inline CONTACT.md:24]:
```suggestion
[eiffel-graphql-api](https://github.com/eiffel-community/eiffel-graphql-api) | [eiffel-graphql-api](https://github.com/orgs/eiffel-community/teams/eiffel-graphql-api) | (mailto:eiffel-graphql-api-maintainers@googlegroups.com)
```

**Review by t-persson — CHANGES_REQUESTED** [review]
Sorry. Completely missed this review.
Created a mail group and added it as a suggested change.

**Review by t-persson — APPROVED** [review]
Thank you

**Pull request merged**